test-packages/e2e-tests/test/bound-function-import-request-builder.spec.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  TestEntity,
+  testService
+} from '@sap-cloud-sdk/test-services-e2e/v4/test-service';
+import { Destination } from '@sap-cloud-sdk/connectivity';
+import nock from 'nock';
+
+describe('bound function import request builder', () => {
+  const { testEntityApi } = testService();
+  const servicePath = TestEntity._defaultServicePath;
+  const host = 'https://example.com';
+
+  const destination: Destination = {
+    url: host,
+    username: 'username',
+    password: 'password',
+    sapClient: '123',
+    authTokens: [],
+    originalProperties: {}
+  };
+
+  it('executes a bound action', async () => {
```

Comment:
[req] I guess this is the location where nock is used.
Not sure whether this is the test that came from the previous unit test?
- if this is an e2e test, then we should use the real e2e app
- if this is a unit test, it should be located in the odata-* package with unit test client

Reply:
Yes I moved the test here thinking this is ok.. I'll look into how to improve it

Reply:
I think this test is a duplicate of the existing e2e tests for bound actions. removed.